docs/source/methodology.md

```diff
@@ -0,0 +1,100 @@
+<!-- docs/source/methodology.md -->
+
+# Methodology
+
+PyDAGMC provides a Pythonic abstraction layer over PyMOAB for interacting with DAGMC (Direct Accelerated Geometry Monte Carlo) `.h5m` files. This section delves into the underlying concepts, data structures, and design choices that enable PyDAGMC's functionality.
+
+## PyDAGMC's Internal Initialization Process
+
+While PyDAGMC is a tool intended for use within a larger, user-defined workflow, it's helpful to understand the internal sequence of operations that occurs when a `.h5m` file is loaded. The following diagram illustrates this initialization process, which culminates in a fully interactive `Model` object ready for querying and manipulation. This is not a prescribed user workflow, but rather a conceptual map of how PyDAGMC parses and prepares the geometry data.
+
+```mermaid
+flowchart TD
+    A["Start: Load .h5m File"] ==> B["Initialize pymoab.Core"]
+    B ==> C["Load Entity Sets"]
+    C ==> D{"CATEGORY_TAG Present?"}
+    D ==> E["Create Objects"]
+    E ==> F["Assign IDs and Tags"]
+    F ==> G["Build Implicit Relationships"]
+    G ==> H["Enable Queries"]
+    H ==> I["Extract Geometry"]
```

Comment:
A lot of this is done as part of MOAB. I don't think it's important to break down a file load in this kind of detail here. Mainly what users will be interested in (in my opinion) is what they can do with a file once loaded. In short, an additional sentence above to capture all of this should suffice. something like:

"When a file is loaded, its embedded geometric topology relationships and metadata (material/property assignments) are loaded according to the conventions established by the DAGMC library."